web/src/components/filings-calendar/tax-access-modal/TaxAccessStepTwo.tsx

```diff
@@ -159,8 +160,27 @@ export const TaxAccessStepTwo = (props: Props): ReactElement => {
           encryptedTaxId: encryptedTaxId as string,
         });
 
+        let userDataWithUpdatedMunicipality = userDataToSet;
+
+        if (userDataWithUpdatedMunicipality.profileData.municipality?.name) {
+          const record = await fetchMunicipalityByName(userDataToSet.profileData.municipality?.name || "");
+
+          userDataWithUpdatedMunicipality = {
+            ...userDataToSet,
+            profileData: {
+              ...userDataToSet.profileData,
+              municipality: {
+                name: record.townName,
+                county: record.countyName,
+                id: record.id,
+                displayName: record.townDisplayName,
```

Comment:
why are we not using the `queueProfileData` that already existed down below here?